apps/nrm/behaviours/get-saved-reports.js

```diff
@@ -37,8 +37,9 @@ const getReports = async(req) => {
 
     // Check the user has an report
     if (savedData) {
+
       // set the session data with previously saved data
-      setSessionData(req, savedData);
+      setSessionData(req, JSON.parse(savedData.json_saved_data));
```

Comment:
erm you are doing JSON.parse on json saved data? 🤔

Reply:
Yeah, it gets stored as a string.

Reply:
ok I guess it's fine.  It just confused me as it's called json so I thought you were passing json but what you mean is your passing a json string